site/public/css/electronic.css

```diff
@@ -1377,6 +1377,17 @@ tr.is_publish {
     padding: 10px;
 }
 
+input[type="radio"][disabled=""]:checked:before {
```

Comment:
Let's stick this definition [here](https://github.com/Submitty/Submitty/blob/1d4866f3c88c333d84db90468409161376f2a22e/site/public/css/server.css#L169) and then it only needs to be defined once across the entire site, in case we decide to use disabled inputs elsewhere.

Reply:
I initially had it as global, but in a discussion with Barb she said that for now we should apply this to only these two instances. Do you still think it is better to have it more globally?

Reply:
Yeah sure, I guess we can cross the bridge of making it global if we add a third place that we have disabled inputs.